src/main/java/atmosfears/AtmosFearsBE/controller/DataIngestController.java

```diff
@@ -0,0 +1,39 @@
+package atmosfears.AtmosFearsBE.controller;
+
+import atmosfears.AtmosFearsBE.model.AirParticulatesFile;
+import atmosfears.AtmosFearsBE.service.AirParticulatesService;
+import atmosfears.AtmosFearsBE.service.CalculationService;
+import atmosfears.AtmosFearsBE.service.DataLoadService;
+import java.util.Set;
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@RequestMapping("/api/ingest")
+public class DataIngestController {
+
+  @Autowired
+  DataLoadService dataLoadService;
+
+  @Autowired
+  CalculationService calculationService;
+
+  @Autowired
+  AirParticulatesService airParticulatesService;
```

Comment:
```suggestion
  private final DataLoadService dataLoadService;

  private final CalculationService calculationService;

  private final AirParticulatesService airParticulatesService;
```
Add `private final` as these variables shouldn't be ever overwritten and should be visible only to this class.

Remove `@Autowired` decorator which is not recommended. You should better use constructor dependency injection whenever you can. To do this, you can either create a constructor by yourself, for example:

```java
public DataIngestController(DataLoadService dataLoadService,
                              CalculationService calculationService,
                              AirParticulatesService airParticulatesService) {
    this.dataLoadService = dataLoadService;
    this.calculationService = calculationService;
    this.airParticulatesService = airParticulatesService;
  }
```

or better, use `@AllArgsConstructor` as a `DataIngestController` class decorator to create this constructor automatically.